### ISSUE TITLE
Incorrect annotation time intervals after running analyze in terminal

### ISSUE THREAD

**Issue by dyang0910** [birdnet-team/BirdNET-Analyzer#847]
**Describe the bug**
Annotation time intervals aren't 3s after training a customized model and running analyze in the terminal.  Instead, we get a time interval that is 3 * speed (the audiospeed we specify in our train parameters).

**To Reproduce**
Training customized model: python -m birdnet_analyzer.train \
          “data/train" \
          --output “/Output“ \
          --test_data "data/test" \
          --hidden_units 256 \
          --dropout 0.75 \
          --batch_size 64 \
          --learning_rate 0.01 \
          --crop_mode "center" \
          --overlap 0.5 \
          --threads 8 \
          --fmin 500 \
          --fmax 8000 \
          --audio_speed 10 \
          --upsampling_mode "repeat" \
          --upsampling_ratio 0.0 \
          --focal-loss-alpha 0.25 \
          --focal-loss-gamma 2.0

Deploy: python -m birdnet_analyzer.analyze "data/test"\
        --classifier "/CustomClassifier.tflite"\
        --output “/Output" \
        --fmin 500 \
        --fmax 8000 \
        --audio_speed 10 \
        --overlap 0.5 

**Expected behavior**
Expected the annotated audio intervals to be 3s, but we get 10s after speeding up the audio 10x. (See screenshot)

**Screenshots**

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/cf37fb99-3ab8-4ff5-92e6-35a16573be46" />

**Desktop (please complete the following information):**
 MacBook Air M1, MacOS 15.7.1; Python 3.11; BirdNET V2.4
